massgov/mayflower
# DP-11078:  Mayflower React: Fuzzy Search Component.

## Description
This creates a component based off InputTextTypeAhead that supports fuzzy search. This uses [Fuse.js ](https://github.com/krisk/fuse)for fuzzy searching. Displaying search and suggestions is handled using react-autowhatever, which is the same that InputTextTypeAhead's AutoSuggest uses under the hood.

## Related Issue / Ticket
- [JIRA issue](https://jira.mass.gov/browse/DP-11078)

## Steps to Test


1. Pull the branch, npm install, start storybook.
2. Browse to Atoms -> forms -> InputTextFuzzy and confirm the component meets the requirements of the ticket.

## Screenshots
![screencapture-localhost-6006-2018-11-05-11_32_16](https://user-images.githubusercontent.com/5257660/48011759-869bdd00-e0ee-11e8-8d43-4db0a0a2d2da.png)
